llvm/lib/Target/AArch64/AArch64FrameLowering.cpp

```diff
@@ -2784,6 +2784,28 @@ struct RegPairInfo {
 
 } // end anonymous namespace
 
+static unsigned findFreePredicateAsCounterReg(unsigned Reg) {
```

Comment:
That function name does not describe what the function does. Consider something like `getAsPnReg()`.